TFT/src/User/Menu/PrintingMenu.c

```diff
@@ -261,6 +261,7 @@ void toggleinfo(void)
       reDrawFan(FAN_ICON_POS);
     }
 
+    storeCmd("M220\nM221\n");
```

Comment:
```suggestion
    speedQuery();
```
Use speedQuery(); to ensure the next query is not added to buffer before the first command is sent, and also reduce code size.

Reply:
I've already tried that in request # 1051
It only works if the hotend and bed are being heated.
As soon as G-codes start to be sent to move the head, the speed icon stops updating.
There is probably a problem with the condition of the speedQuery ();

Reply:
@guruathwal  you have a suggestion on how to modify speedQuery (); to work in this case as well?

Reply:
@guruathwal your change proposal only works if I also adjust speedQuery this way. Do you agree?

![image](https://user-images.githubusercontent.com/46979052/92417007-5d23c080-f160-11ea-949c-73c76fb1404f.png)

Reply:
I will check and revert to you.

Reply:
@radek8 the implementation of infohost.wait needs work

Reply:
I don't understand your answers.
under the condition (infoHost.wait==false), updating the print speed value does not work correctly.
Do you know another way to solve it?